src/test/java/io/bf2/kafka/authorizer/CustomAclAuthorizerTest.java

```diff
@@ -71,7 +79,7 @@ void setup() {
                 return results;
             });
 
-        Mockito.when(this.delegate.acls(Mockito.any(AclBindingFilter.class)))
+        Mockito.when(this.delegate.acls(any(AclBindingFilter.class)))
```

Comment:
Add static import for `when`?

Reply:
Done, I'm a little unsure if there is some convention around static imports the test is full of `Mockito.mock`, `Mockito.any*` calls.

Reply:
I don't think there is an established convention, but for me the `Mockito` prefix is largely noise in tests (and the inconsistent usage annoys me even more).